Sources/SwiftPhoenixClient/Defaults.swift

```diff
@@ -38,22 +38,22 @@ public class Defaults {
   public static let rejoinSteppedBackOff: (Int) -> TimeInterval = { tries in
     return tries > 3 ? 10 : [1, 2, 5][tries - 1]
   }
-  
+
+  public static let version: SerializerVersion = .v1
```

Comment:
I'd prefer naming this `vsn` to match the naming pattern in phoenix and setting the default to `.v2`

This, with the changes to `payload` will go out in a major release so I'm comfortable setting the default to v2

Reply:
@dsrees since it's going to be a breaking change maybe it's ok to drop `.v1` completely to simplify the logic? `vsn=2.0.0` has been supported since [phx-1.3](https://github.com/phoenixframework/phoenix/blob/v1.3/lib/phoenix/transports/v2/websocket_serializer.ex) which [came out ~4 years ago](https://github.com/phoenixframework/phoenix/blob/v1.3/CHANGELOG.md#130-2017-07-28) so it's unlikely anyone using this client would ever need to pick `.v1`.

Reply:
`phoenix.js` doesn't use v1 version even if `vsn=1.0.0` is provided in options, it still sends messages formatted for `v2` and makes server crash.

Reply:
Yeah i think we can drop it

I'm also leaning towards letting `vsn` be a `String`, which allows a developer to provide a custom value. I'm assuming that was the logic behind the decision to leave it as an `opt` in the js library. Thoughts?

Reply:
@dsrees it's hard for me to imagine a scenario where `vsn` would be set to anything other than the default `2.0.0`. But following `phoenix.js` implementation is probably the safest approach, I'll update the PR to remove `SerializerVersion` and make `vsn` a string.

Reply:
My thoughts exactly. Thanks